src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -314,16 +326,144 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
                 = OPP_AutomatedSoftCreditsService.createAffiliationOCRs(accountsToOpportunities, transactionOCRs);
             dmlWrapper.objectsToInsert.addAll(affiliationOCRs);
         }
+        
+        //For multiple tribute, we need to process the DML here since Tribute trigger will not call run method
+        if(processDML) {
+            TDTM_TriggerHandler.disableTDTM = true;
```

Comment:
@weizsforg - Do you remember why you did this?  I am not sure of the exact purpose in this instance.

Reply:
Oh, because we call the upsertContactRolesAfterInsert from TBU_OpportunityContactRoles_TDTM class. Since TBU_OpportunityContactRoles_TDTM does not call the "Run" method, we need to process the DML right away in this method. If the logic run in
TBU_OpportunityContactRoles_TDTM, we don't want it to trigger other event like Opp TDTM to Create/Update OCR again.